sn06.yml

```diff
@@ -87,6 +87,44 @@
     - name: Disable SELinux
       selinux:
         state: disabled
+    - name: Inject custom Pulsar build in the Galaxy requirements file (HTCondor migration).
+      # This Pulsar build makes the `condor_rm` and `condor_submit` commands configurable.
+      # See https://github.com/kysrpex/pulsar/commits/condor_manager_prefix_option for more details.
+      block:
+        - name: Allocate a temporary directory.
+          ansible.builtin.tempfile:
+            prefix: ansible.galaxyproject.galaxy_requirements_file
+            state: directory
+          changed_when: false
+          register: galaxy_requirements_file_directory
+        - name: Clone Galaxy.
+          git:
+            dest: "{{ galaxy_requirements_file_directory.path }}/galaxy"
+            depth: 1
+            repo: "{{ galaxy_repo }}"
+            version: "{{ galaxy_commit_id }}"
+            executable: "{{ git_executable | default(omit) }}"
+          changed_when: false
+        - name: Retrieve the requirements file.
+          ansible.builtin.copy:
+            remote_src: true
+            src: "{{ galaxy_requirements_file_directory.path }}/galaxy/lib/galaxy/dependencies/pinned-requirements.txt"
+            dest: "{{ galaxy_requirements_file_directory.path }}/requirements.txt"
+          changed_when: false
+        - name: Replace `pulsar-galaxy-lib` with a patched version in the requirements file.
+          ansible.builtin.lineinfile:
+            path: "{{ galaxy_requirements_file_directory.path }}/requirements.txt"
+            regexp: 'pulsar-galaxy-lib'
+            line: 'git+https://github.com/kysrpex/pulsar.git@condor_manager_prefix_option#egg=pulsar-galaxy-lib ; python_version >= "3.7" and python_version < "3.12"'
```

Comment:
To see what's changed, click [here](https://github.com/kysrpex/pulsar/compare/master...kysrpex:condor_manager_prefix_option).